### PR TITLE
End of the world backups

### PR DESCRIPTION
**Pull request by ahoneybun**
This is a start of this article though I wanted to open a PR to start the discussion with @storyaddict to work on this new article.

### CODE REVIEW

**Comment by ahoneybun** [comment]
I added some new content addressing your comments.

File: _articles/disaster-recovery.md [hunk 1]
```diff
@@ -57,3 +57,7 @@ Once we have our data backed up we can reinstall the OS if we are ready to do th
 
 ![Restoring data in Deja-Dup](/images/backup/deja_overview.png)
 
+## Future Steps
+
+Now that we have our data and a clean install of the OS let's prepare so that we will be ready if this issue happens again using our backup article [here](/articles/backup-files).
```
Comment (storyaddict) [inline _articles/disaster-recovery.md:62]: Sentence adjustments:

Now that we have our data, and a clean install of the OS, let's prepare for if this issue happens again. It's good practice to set up scheduled backups. Our backup article can help:

**Review by storyaddict — CHANGES_REQUESTED** [review]

**Review by storyaddict — CHANGES_REQUESTED** [review]
please see latest commit :)

**Comment by ahoneybun** [comment]
Commit? What changes do we need to do or add?

**Review by storyaddict — APPROVED** [review]
Changes look good!

**Pull request merged**